### DISCUSSION TITLE
Welcome to MapStruct Discussions!

### DISCUSSION THREAD

**Discussion by filiphr** [mapstruct/mapstruct#2518] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Share ideas.
  * Engage with other community members.
  * Ask questions you’re wondering about. We prefer questions here to be mostly about our codebase, how you can contribute, directions for new features, new ideas, etc. If you have questions about using MapStruct please continue using [StackOverflow](https://stackoverflow.com/questions/tagged/mapstruct) 🙏 .
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

We still do have the [mapstruct-users](https://groups.google.com/g/mapstruct-users/) group. However, we believe that using GitHub discussions would be easier for us maintainers to keep you updated and to have a bigger interaction with the community.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Chessray** [comment]
OK, I'll be the first:

My name is Raimund. I'm originally from Germany, but have been in the UK since late 2013 where I've been running a [consulting company](https://www.kleindelao.co.uk/) for a little over 3 years now. My creed has been for a very long time and still is: I'm whatever a project needs me to be. I write code, draw boxes and lines, run the Scrum ceremony, mentor juniors, work with the client,...

Outside work, our daughter takes up most of my time with all her sports - Modern Pentathlon is a logistical challenge for parents. In the little time that leaves me, you may find me at the chessboard or on the baseball field - yes, that _is_ a thing in the UK.

As for MapStruct, I've been using it on and off in various projects for more than five years now, and I'm still enjoying it. Nearly two years back I had the idea of somehow [combining MapStruct mappers with Spring's ConversionService](https://stackoverflow.com/questions/58081224/mapstruct-mapper-as-spring-framework-converter-idiomatic-use-possible). What followed, was a fruitful discussion on the Gitter channel ending in @sjaakd's idea of adding another generated Spring component acting as a bridge.

A few months later I had some time at my hands to put this idea into practice. Sjaak and Filip were still very much on board with it, and before knowing I found myself in charge of MapStruct's Spring-specific [extensions project](https://github.com/mapstruct/mapstruct-spring-extensions).

I suppose that's it for the purpose of this thread. Please feel free to contact me with anything you'd like to ask about what I mentioned. 🙂

**Comment by Zegveld** [comment]
Might as well join in:

My name is Ben. I'm from the Netherlands. At the company that I'm currently working for I fill the job as both lead software engineer, and also the software architect for the projects that land with the team I'm in.

Outside work, I enjoy playing RPG's both PnP-style as well as the computer game variant. In the past I've enjoyed playing chess, but these days you'll find me once every 2 or 3 days in a bouldering hall, climbing walls of up to 4.5meter tall.

As for mapstruct, I've done several small fixes. Also got a feature pending for the 1.5.0 release: [Support for subclass mappings](https://github.com/mapstruct/mapstruct/issues/131). You might see me pop-up with PR's that contain solutions for some of the issues.

I guess that is it for the purpose of this thread. If you want to know something, feel free to contact me. (although I might be slow to respond from time to time)
